.golangci.yml

```diff
@@ -117,9 +117,9 @@ linters:
     - deadcode
     - depguard
     - errcheck
-    - funlen
+    # - funlen
     - goconst
-    - gocyclo
+    # - gocyclo
```

Comment:
If you're commenting those out, you should remove them completely, or fix the issues they highlighted, or put in a `nolint` rule on that line with an issue.